project.yaml

```diff
@@ -66,15 +66,15 @@ targets:
       x86_64:
         test_distros:
           representative:
-          - ubuntu-os-cloud:ubuntu-2004-lts
+          - ubuntu-os-cloud:ubuntu-2404-lts-amd64
```

Comment:
Why are we changing the `focal` section, rather than adding a new `noble` one?

As-is, these changes should probably be reverted after testing…

Reply:
We don't have the Kokoro machinery up yet; our dev docs recommend doing it this way temporarily (including the revert before merging).